r55/src/exec.rs

```diff
@@ -44,7 +44,7 @@ pub fn run_tx(db: &mut InMemoryDB, addr: &Address, calldata: Vec<u8>) {
     let mut evm = Evm::builder()
         .with_db(db)
         .modify_tx_env(|tx| {
-            tx.caller = address!("0000000000000000000000000000000000000007");
+            tx.caller = address!("0000000000000000000000000000000000000001");
```

Comment:
why the change back?

Reply:
I guess this will all move into more structured unit tests anyway

Reply:
Added multiple addresses to test so started from `0x..01`, nothing major.

> I guess this will all move into more structured unit tests anyway

Yeah, in fact I might as well remove it from here to keep it clean.